zict/func.py

```diff
@@ -74,13 +74,6 @@ def __delitem__(self, key: KT) -> None:
     def keys(self) -> KeysView[KT]:
```

Comment:
nit: Can we also drop this? IIUC, this should be covered by the base implementation.

Reply:
yes